app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+
+    # Assign the comment to the current_user
+    @comment.user = current_user
+
```

Comment:
- [ ] In the process of creating a comment for a given post for the `current_user`, kindly ensure that comment is successfully created for the current_user. 🙏